### PR TITLE
Improve scibite cord transform

### PR DESCRIPTION
**Pull request by deepakunni3**
This should fix the transform such that the output nodes and edges are entirely CURIEs.

### CODE REVIEW

**Comment by deepakunni3** [comment]
Switching to draft. Need some more fine tuning

**Comment by deepakunni3** [comment]
@justaddcoffee Thanks for pointing that out

**Comment by justaddcoffee** [comment]
NP! Oh, also, need to add that block I took out back to `merge.yaml`: 
```
   SciBite-CORD-19:
      type: tsv
      filename:
         - data/transformed/SciBite-CORD-19/nodes.tsv
         - data/transformed/SciBite-CORD-19/edges.tsv
```

File: kg_covid_19/transform_utils/scibite_cord/scibite_cord.py [hunk 1]
```diff
@@ -263,6 +284,125 @@ def extract_termite_hits(self, data: Dict) -> Set:
...
+            if identifier in self.gene_info_map:
+                curie = f"NCBIGene:{self.gene_info_map[identifier]['NCBI']}"
+        else:
+            if self.is_iri(iri):
+                curie = contract_uri(iri)
+                if curie:
+                    curie = curie[0]
+                else:
+                    curie = contract_uri(iri, cmaps=[CUSTOM_CMAP])
+                    print(f"Contracting URI {iri} to {curie} via custom map")
```
Comment (justaddcoffee) [inline kg_covid_19/transform_utils/scibite_cord/scibite_cord.py:312]: One more request - could we omit or making this into a `logging.DEBUG()` call instead? I'm running the ingest and it's a little chatty

**Comment by justaddcoffee** [comment]
commit 7d808cbef1a2eee6209cef274cbd314025ed2056 runs great and to completion on my laptop

**Comment by justaddcoffee** [comment]
Looks good @deepakunni3 ! If possible, could we use a tempdir to unzip files, so `data/raw` stays pristine, so to speak? 

I'll send a diff you can `git apply` on gitter, if you want

**Pull request merged**